crates/matrix-sdk/src/send_queue.rs

```diff
@@ -1171,8 +1175,8 @@ pub enum LocalEchoContent {
         /// A handle to manipulate the sending of the associated event.
         send_handle: SendHandle,
         /// Whether trying to send this local echo failed in the past with an
-        /// unrecoverable error (see [`SendQueueRoomError::is_recoverable`]).
-        is_wedged: bool,
+        /// error (see [`SendQueueRoomError::is_recoverable`]).
```

Comment:
Let's keep `unrecoverable` here. It has a precise meaning in the context of this file, i.e. retrying would lead to the same error again and again (vs network errors that may be transient).